scripts/globals/effects/confrontation.lua

```diff
@@ -8,6 +8,10 @@ function onEffectGain(target,effect)
     if (target:getPet()) then
         target:getPet():addStatusEffect(effect)
     end
+
+    if player:getObjType() == tpz.objType.PC then
+        target:clearTrusts()
+    end
```

Comment:
Should this be added to the battlefield status as well?

Reply:
Also these should be `target:getObjType()` not `player:getObjType()`

Reply:
Good catch, thanks!